packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -147,3 +147,35 @@ function valueToAttributeValue(
       return {};
   }
 }
+
+const HTTP_ATTRIBUTE_MAPPING: {[key: string]: string} = {
+  'http.method': '/http/method',
+  'http.url': '/http/url',
+  'http.target': '/http/target',
+  'http.host': '/http/host',
+  'http.scheme': '/http/client_protocol',
+  'http.status_code': '/http/status_code',
+  'http.flavor': '/http/flavor',
+  'http.user_agent': '/http/user_agent',
+  'http.request_content_length': '/http/request_content_length',
+  'http.request_content_length_uncompressed':
+    '/http/request_content_length_uncompressed',
+  'http.response_content_length': '/http/response_content_length',
+  'http.response_content_length_uncompressed':
+    '/http/response_content_length_uncompressed',
+  'http.server_name': '/http/server_name',
+  'http.route': '/http/route',
+  'http.client_ip': '/http/client_ip',
+  'http.path': '/http/path',
```

Comment:
This one's not in https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes